android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -63,11 +65,21 @@ public void initNativeSdk(final PluginCall call) {
         SentryAndroid.init(
             this.getContext(),
             options -> {
+                SdkVersion sdkVersion = options.getSdkVersion();
+                if (sdkVersion == null) {
+                    sdkVersion = new SdkVersion(ANDROID_SDK_NAME, BuildConfig.VERSION_NAME);
+                } else {
+                    sdkVersion.setName(ANDROID_SDK_NAME);
+                }
+
                 if (capOptions.has("debug") && capOptions.getBool("debug")) {
                     options.setDebug(true);
                     logger.setLevel(Level.INFO);
                 }
 
+                options.setSentryClientName(sdkVersion.getName() + "/" + sdkVersion.getVersion());
+                options.setNativeSdkName(NATIVE_SDK_NAME);
+
```

Comment:
`setSdkVersion` should be called to cover the case when the `sdkVersion` is null.

```suggestion
            options.setSdkVersion(sdkVersion);

```